src/modules/organizations/containers/details-form-container.jsx

```diff
@@ -93,14 +95,16 @@ class DetailsFormContainer extends React.Component {
           <fieldset className="form__fieldset">
             <label className="form__label" htmlFor="introduction">
               Introduction
-              <MarkdownEditor
-                id="introduction"
-                name="introduction"
-                onChange={this.handleTextAreaChange}
-                project={this.props.organization}
-                rows="10"
-                value={this.state.textarea}
-              />
+              <div className="form__input--top-margin">
```

Comment:
Not a big problem, but I don't think `div` is a valid child of `label` in HTML. If you need a generic wrapper inside a label, use `span`.

If this div is just here in order to apply the CSS class to the Markdown editor, you can add that directly to the `MarkdownEditor` component.